contrib/awssqs/pkg/apis/sources/v1alpha1/aws_sqs_types_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Knative Authors
+Copyright 2019 The Knative Authors
```

Comment:
is that 2018-2019  ?   (sorry if that might been answered before)

cc @vaikas-google

Reply:
I could not find any 2018-2019 in the tree so I assumed latest year. But let me know the correct one :)

Reply:
I believe that the latest year is sufficient.

Reply:
If you are super-interested, I can try to catch one of our lawyercats for more policy.